App/Seller/Sources/Seller/UI/MessagesView/MyMessagesView.swift

```diff
@@ -0,0 +1,194 @@
+// system
+import SwiftUI
+
+// Internal
+
+// 3rd
+import OptionalAPI
+
+
+protocol SizingFrameType {
+    
+    static var instance: SizingFrameType { get }
+    
+    var size: CGSize { get }
+}
+
+#if os(macOS)
```

Comment:
Do you have `main` merged in as I can see this change already exists.

Reply:
Weird, main should have been (and imo was) merged :D

Reply:
Checked again, rebased xD